packages/common/src/types/userWarehouseCredentials.ts

```diff
@@ -36,7 +36,12 @@ export type UserWarehouseCredentialsWithSecrets = Pick<
         | Pick<CreatePostgresCredentials, 'type' | 'user' | 'password'>
         | Pick<
               CreateSnowflakeCredentials,
-              'type' | 'user' | 'password' | 'authenticationType' | 'token'
+              | 'type'
+              | 'user'
+              | 'password'
+              | 'authenticationType'
+              | 'refreshToken'
+              | 'token'
```

Comment:
I kept `token` here so we can check it here https://github.com/lightdash/lightdash/pull/18919/changes#diff-31e83ba5002599e953eb94d38816460bf2352d4651e2836c0757854a00e05592R198